doc/nrf/release_notes.rst

```diff
@@ -8,10 +8,10 @@ See the release notes for information about specific |NCS| releases.
 Important issues that are discovered after a release is tagged are listed on the :ref:`known_issues` page.
 
 .. important::
-   A "99" at the end of the version number of this documentation indicates continuous updates on the master branch since the previous major.minor release.
-   Changes between releases that are merged into the master branch are tracked in the `nRF Connect SDK changelog`_ in the sdk-nrf repository wiki.
+   A "99" at the end of the version number of this documentation indicates continuous updates on the master branch since the previous major or minor release.
```

Comment:
This was actually correct. The release number is major.minor.bugfix.

Reply:
Right, my bad.